src/components/calcite-date-day/calcite-date-day.scss

```diff
@@ -0,0 +1,7 @@
+:host,
+:host(:active),
+:host(:focus),
+:host(:hover),
+:host(:visited) {
+  outline: none;
```

Comment:
should we be doing this? :)

Reply:
We have a 2px stroke for Active, it's mainly noticed on the range:

![image](https://user-images.githubusercontent.com/6144462/62799090-bc625d80-ba94-11e9-879d-205716d11dff.png)